D/Dbus/build_tarballs.jl

```diff
@@ -33,18 +31,19 @@ make install
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = filter!(p -> Sys.islinux(p) || Sys.isfreebsd(p), supported_platforms())
-# TODO: Remove this restriction for the next build
-filter!(p -> arch(p) != "armv6l", platforms)
 
+# The products that we will ensure are always built
 products = [
     LibraryProduct("libdbus-1", :libdbus),
     ExecutableProduct("dbus-daemon", :dbus_daemon)
 ]
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("Expat_jll"; compat="2.2.7"),
+    Dependency(PackageSpec(name="Expat_jll", uuid="2e619515-83b5-522b-bb60-26c02a35a201"); platforms=platforms),
+    BuildDependency(PackageSpec(name="Xorg_xorgproto_jll", uuid="c4d99508-4286-5418-9131-c86396af500b"); platforms=platforms),
+    BuildDependency(PackageSpec(name="Xorg_libX11_jll", uuid="4f6342f7-b3d2-589e-9d20-edeb45f2b2bc"); platforms=platforms),
```

Comment:
This should be a normal dependency - it provides a shared library to link against.

Also, if a dependency is needed for all platforms, then don't specify the `platforms` argument to it.

Reply:
It looks like the original script _did not_ include `X11` support. Given that leaving `--with-x` would mean `--with-x=auto`, then it would not detect `X11` and just "jump over". Checking `ldd` output on both artifact on `v"1.12.6"` confirms that there is no link to it.

On the new version, `ldd` outputs the _same_ for both artifacts. Meaning that `X11` support is for some other generated file, not for the ones we use. I'll set `--with-x=no` and delete the dependencies.

Reply:
BTW, the change in behavior happened [here](https://gitlab.freedesktop.org/dbus/dbus/-/merge_requests/263).